.github/workflows/hardware-long.yml

```
@@ -29,6 +29,9 @@ jobs:
           - board: p150a
             runs-on:
               - p150a-jtag
+          - board: p300a
+            runs-on:
+              - p300-jtag
```

```suggestion
              - p300a-jtag
```
This follows how we tag other runners (and is also the name I used)